plugins/jobtype/src/azkaban/jobtype/HadoopSecureSparkWrapper.java

```diff
@@ -16,10 +16,8 @@
 
 package azkaban.jobtype;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.lang.reflect.Field;
+import azkaban.utils.Props;
```

Comment:
Imports are already organised in second commit. I will add blank lines between each group of imports.